src/LocalDate.js

```diff
@@ -82,6 +82,22 @@ const  DAYS_0000_TO_1970 = (DAYS_PER_CYCLE * 5) - (30 * 365 + 7);
 
 export class LocalDate extends ChronoLocalDate{
 
+    /**
+     * The minimum supported {@link LocalDate}
+     * This could be used by an application as a "far past" date.
+     *
+     * @type {LocalDate}
+     */
+    static MIN;
```

Comment:
Hm... where is the definition of `MIN` ... do we still need the `_init` trick? or can we define the statics here ?

Reply:
Yes, unfortunatly we still need the `_init` trick. 

The benefit is that we have nicer esdoc with an extra `Static Member Summary` section, with the downside that we have that code without a definition. 

Well, i am not sure, if we should introduce static members unless we solve the dependency cycle thingy...